applications/FluidDynamicsApplication/custom_utilities/embedded_discontinuous_data.h

```diff
@@ -46,6 +49,8 @@ double SlipLength;
 double PenaltyCoefficient;
 
 NodalScalarData ElementalDistances;
+Vector ElementalEdgeDistances;
+Vector ElementalEdgeDistancesExtrapolated;
```

Comment:
Couldn't we make these a `BoundedVector<double, NumEdges>` (size is known at compile time).